src/sentry/sentry_metrics/multiprocess.py

```diff
@@ -98,7 +98,7 @@ class MetricsBatchBuilder:
     def __init__(self, max_batch_size: int, max_batch_time: float) -> None:
         self.__messages: MessageBatch = []
         self.__max_batch_size = max_batch_size
-        self.__deadline = time.time() + max_batch_time
+        self.__deadline = time.time() + max_batch_time / 1000.0
```

Comment:
(I think you said somewhere it's a known issue @MeredithAnya ?)

Ideally, we should probably do the millisecond -> second translation on a higher abstraction level, e.g. inside or around `get_streaming_metrics_consumer` https://github.com/getsentry/sentry/blob/master/src/sentry/runner/commands/run.py#L567